source/sass/buyers-guide/views/product.scss

```diff
@@ -196,23 +196,13 @@
   content: ' '; /* :before and :after won't work without content... =( */
   position: absolute;
   top: 0;
-  left: -100%;
-  right: -100%;
+  left: 0;
+  right: 0;
   height: 228px;
   background-image: url("/_images/buyers-guide/dot-pattern.svg");
   background-size: 9px 9px;
 }
 
-.row.with-product-updates {
```

Comment:
I think these might still be necessary; this section looks pretty narrow on mobile.

Reply:
(I'm not sure this is actually the solution, but my "request changes" is to fix that in general)

Reply:
Ummmm it's weird. It's exactly the same code but things look differently on my machine and on review app. I'll investigate.

# my machine
![screen shot 2018-11-29 at 12 45 08](https://user-images.githubusercontent.com/2896608/49250821-b92ab400-f3d4-11e8-88db-50dda6405806.png)

# Heroku review app
![screen shot 2018-11-29 at 12 45 12](https://user-images.githubusercontent.com/2896608/49250819-b92ab400-f3d4-11e8-90a4-c394777f9fa3.png)

Reply:
Ah it's something to do with the "compare products" section. I'll update my PR.

Reply:
I think it's something to do with how other sections on this page are set up (e.g., with/without product image, compare products section etc). I'll take a closer look in a bit when I get to the office.